Examples/StylesAndOptions/Example/ChooseStyle.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ChooseStyle.swift
+//  Example
+//
+//  Created by Nataliya Patsovska on 2018-06-12.
+//  Copyright © 2018 iZettle. All rights reserved.
+//
+
+import Flow
+import Presentation
+
+struct ChooseStyle {
+    private let createDataSource: (UIView) -> DataSource<PresentationStyle>
+
+    init() {
+        createDataSource = { view in
+            let styles: [PresentationStyle] = [
+                .default,
+                .modal,
+                .embed(in: view),
+                .invisible,
+                .sheet(from: view, rect: nil),
+                .popover(from: view),
+            ]
+            return DataSource(options: styles)
+        }
+    }
+}
+
+extension ChooseStyle: Presentable {
+    typealias ChooseStyleResult = (PresentationStyle, preferredPresenter: UIViewController?, alertToPresent: Alert<()>?)
```

Comment:
Trying to follow we can't just signal the style here? 
- Not sure why we need `preferredPresenter`? What would happen if we pass nil for "embed"?
- The `alertToPresent` seems to be used by "sheet" and uses a constant value, so this check could be moved to the other end?